.env

```diff
@@ -1,4 +1,4 @@
 REACT_APP_API_URL = "http://localhost:1337"
-
+REACT_APP_API_DOCUMENTATION_URL=https://zoonotify-dev.bfr.berlin/cms/documentation/v1.0.0
```

Comment:
The BASE URL is already set correctly for every environment - no need to set this here again